angular/src/app/containers/leaflet/leaflet.component.ts

```diff
@@ -29,11 +47,76 @@ export class LeafletComponent implements AfterViewInit {
         bounds: config.maxBounds
       });
 
-      baseLayer.addTo(leafletMap);
+      baseLayer.addTo(this.leafletMap);
 
       setTimeout(() => {
-        leafletMap.invalidateSize();
+        this.leafletMap.invalidateSize();
       });
+
+      this.leafletMap.on('moveend', this.updateBoundingBox.bind(this));
```

Comment:
Kunnen we dit niet beter via een reduceren laten verlopen? Klinkt misschien als een ietwat overdreven abstractie, maar dan is het wel gelijk in lijn met de beoogde architectuur. Anderzijds integreert het misschien weer wat lastig met Leaflet. Wat vind jij?